app/src/main/java/fr/neamar/kiss/SettingsActivity.java

```diff
@@ -320,7 +320,9 @@ public void onPause() {
     private void fixSummaries() {
         int historyLength = KissApplication.getDataHandler(this).getHistoryLength();
         if (historyLength > 5) {
-            findPreference("reset").setSummary(String.format(getString(R.string.items_title), historyLength));
+            Preference resetScroll = findPreference("resetScroll");
```

Comment:
I'm super late to the party, but while refactoring code I noticed that you refer to `resetScroll`. This preference doesn't exist, and even if it did I don't understand why we'd display the history here? For now I'm going to assume this is a mistake and revert this change :)

Reply:
I think the IDE changed it from reset to resetScroll while I was changing things.